### ISSUE TITLE
Members section UI

### ISSUE THREAD

**Issue by ghost** [UCSDTT/UCSDTTWebsite#5]
The Members section grid suffers from a [grid illusion](https://en.wikipedia.org/wiki/Grid_illusion). This issue arises from the high contrast between the vignetted photos and the white background.

An improvement would be to rethink the design or color scheme.

**Comment by ghost** [comment]
Notice the fleeting grey blur that appears between the intersection of the white grid lines. Illusion might not be reproducible in all displays but I remember people bringing it up before.

![screen shot 2016-02-16 at 1 40 20 pm](https://cloud.githubusercontent.com/assets/4935325/13092037/d71a1030-d4b2-11e5-826c-e3969e4ee815.png)

**Comment by bgenchel** [comment]
This one is definitely an optical illusion. I'm not sure there's anything
we could do for you there.

**Comment by jezhou** [comment]
We could put the images closer together, and do a little stylizing. Apparently, that [lessens the effect](http://ux.stackexchange.com/questions/53791/avoiding-the-hermann-grid-illusion). 

Also did not know about this before. Learning something new every day :3

**Comment by alxyang** [comment]
can't unsee

**Comment by michael-luo** [comment]
was this ever addressed lol

**Comment by bgenchel** [comment]
I don't even know who's in charge of this anymore lol

**Comment by jezhou** [comment]
@michael-luo no it wasn't
@bgenchel i am, but there's a whole can of worms that we need to fix now. there's a request for a redesign (which is done on paper) and we're trying to move to a more modern framework like react (to give the younger members a chance to learn it, and because jade [now pug] is very outdated at this point)

we will probably be moving to another repository as well

**Comment by michael-luo** [comment]
Yeah totally understand. I would feel more comfortable redesigning it too if I was working on this project. Are you all using React because you want to learn it or because it's suitable for a project of this scope? If the younger members want to learn it and if React is overkill, then might as well start a different side project and use React there right? Maybe build some Twitch chat bots
